src/Actions/babylon.actionManager.ts

```diff
@@ -138,9 +143,8 @@
             return ActionManager._OnKeyUpTrigger;
         }
 
-        public static DragMovementThreshold = 10; // in pixels

```

Comment:
Well..actually I'm wrong..You are right here. The DragMovementThreshold  will not only affect ActionManager but all pointer observable..
So this change makes sense. Can you just add it in the what's new section about breaking changes?